aerospike-core/Cargo.toml

```diff
@@ -8,13 +8,13 @@ edition = "2018"
 [dependencies]
 log = "0.4"
 byteorder = "1.3"
-ripemd160 = "0.8"
-base64 = "0.11"
-crossbeam-queue = "0.2"
-rand = "0.7"
+ripemd = "0.1"
```

Comment:
ripemd160 was not maintained last time I checked and we were getting audit warnings